File: 250721/좀 더 어려운 수학 점수/math-scores-are-more-difficult.py
# 입력
intAmath, intAeng = map(int, input().split())
intBmath, intBeng = map(int, input().split())

# 조건문
if intAmath > intBmath :
    print("A")
elif intAmath == intBmath :
    if intAeng > intBeng :
        print("A")
    else :
        print("B")
else :
    print("B")

File: 250721/좀 더 어려운 수학 점수/math-scores-are-more-difficult.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int intAmath = sc.nextInt();
        int intAeng = sc.nextInt();
        sc.nextLine();
        int intBmath = sc.nextInt();
        int intBeng = sc.nextInt();

        // 변수 정의
        char charName;

        // 조건문
        if (intAmath > intBmath) {
            charName = 'A';
        } else if (intAmath == intBmath) {
            if (intAeng > intBeng) {
                charName = 'A';
            } else {
                charName = 'B';
            }
        } else {
            charName = 'B';
        }

        // 출력
        System.out.println(charName);
    }
}